sympy/printing/tests/test_fcode.py

```diff
@@ -89,6 +89,14 @@ def test_case():
     assert ob.doprint(X*sin(x) + x_, assign_to='me') == '      me = X*sin(x_) + x__'
     assert ob.doprint(X*sin(x), assign_to='mu') == '      mu = X*sin(x_)'
     assert ob.doprint(x_, assign_to='ad') == '      ad = x__'
+    A = IndexedBase('A')
+    i = Idx('i', symbols('n', integer=True))
```

Comment:
could we have a test with two indices, e.g. `i` and `I` to make (really) sure name mangling works for instances of `Idx`?